mesheryctl/internal/cli/root/system/start.go

```diff
@@ -105,6 +105,9 @@ func start() error {
 		return err
 	}
 
+	if utils.PlatformFlag == "" {
+		utils.PlatformFlag = currCtx.GetPlatform()
+	}
 	if utils.PlatformFlag == "docker" || utils.PlatformFlag == "kubernetes" {
 		currCtx.SetPlatform(utils.PlatformFlag)
 	} else {
```

Comment:
better to cover these if-else, in another if statement
 `if utils.PlatformFlag != "" {}`

Reply:
Piyush, let’s get this fixed urgently. Will you “suggest” this fix with CTRL+G?

Reply:
@leecalcote @piyushsingariya I made the final changes. Can u check once.